README.md

```diff
@@ -28,8 +28,12 @@ gulp.task('css', function() {
 #### ruleConfiguration
 Type: `Object`
 
+If you pass `lookup: false`, the local .csslintrc is not looked up automatically.
```

Comment:
I suppose that, if there is ever a CSSLint property called `lookup`, we'll have a problem...